CHANGELOG.md

```diff
@@ -2,6 +2,38 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.5.0](https://github.com/aws/aws-rfdk/compare/v1.4.0...v1.5.0) (2024-12-05)
+
+Note that the default versions of DocumentDB and MongoDB have been updated. Make sure to backup your databases before updating to RFDK 1.5. See our documentation on [upgrading to RFDK 1.5](https://github.com/aws/aws-rfdk/blob/v1.5.0/packages/aws-rfdk/docs/upgrade/upgrading-1.5.md) for more information.
+
+
+### Supported CDK Version
+
+* [2.163.0](https://github.com/aws/aws-cdk/releases/tag/v2.163.0)
+
+
+### Officially Supported Deadline Versions
+
+* [10.1.9.2 to 10.4.0.10](https://docs.thinkboxsoftware.com/products/deadline/10.4/1_User%20Manual/manual/release-notes.html)
+
+
+### ⚠ BREAKING CHANGES
+
+* Upgrades MongoDB version to 8.0 (#1392)
+* Upgrades DocumentDB version to 5.0.0 (#1364)
```

Comment:
I understand that deployment will fail unless you manually upgrade your existing database?  We should probably call that out explicitly here.